web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import { Typography } from '@material-ui/core';
+import "./ViewFeedbackResponses.css";
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import AvatarComponent from '../avatar/Avatar'
+import MoodBadIcon from '@material-ui/icons/MoodBad';
+import InsertEmoticonIcon from '@material-ui/icons/InsertEmoticon';
+import SentimentSatisfiedIcon from '@material-ui/icons/SentimentSatisfied';
+//note that request id will be in actual object, so you will need to get information out of request id, template id and state
+
+
+const useStylesCardContent = makeStyles({
+    root: {
+      '&:last-child': {
+        paddingBottom: '16px',
+      }
+    }
+  }, { name: "MuiCardContent" })
+
+
+
+
+const ViewFeedbackResponses = (props) => {
+     useStylesCardContent();
```

Comment:
I believe this needs assignment to a variable to use it in the component.

Reply:
It actually complains if I assign the styles hook to a variable and never use that variable. Just calling the hook in a component is enough to override the default Material UI classes that it attached to the card component automatically.